data-raw/UN_WPP_2022_life_expectancy_Africa.R

```diff
@@ -0,0 +1,11 @@
+## Life expectancy
+# UN WPP
+# Life expectancy at Exact Age x (ex) - Both Sexes
+# The average number of remaining years of life expected by a hypothetical
+# cohort of individuals alive at age x who would be subject during the remaining
+# of their lives to the mortality rates of a given year. It is expressed as years.
+# https://population.un.org/wpp/Download/Standard/Mortality/
```

Comment:
This life expectancy source makes sense to use-- may be obvious, but worth noting explicitly that this DALY functionality should not be used when modelling countries outside of Africa